x-pack/plugins/monitoring/public/components/beats/overview/overview.js

```diff
@@ -73,10 +74,9 @@ function renderLatestActive(latestActive, latestTypes, latestVersions, intl) {
     );
   }
 
-  const calloutMsg = intl.formatMessage({
-    id: 'xpack.monitoring.beats.overview.noActivityDescription',
+  const calloutMsg = i18n.translate('xpack.monitoring.beats.overview.noActivityDescription', {
     // eslint-disable-next-line max-len
```

Comment:
@maryia-lapata Is there a benefit of using one over the other?

I personally think the one-liner with lint bypass is cleaner (since it's a pre-existing change)